package-managers/qubes-download-dom0-updates.sh

```diff
@@ -84,7 +84,7 @@ rpm --root=$DOM0_UPDATES_DIR --rebuilddb
 
 if [ "$CLEAN" = "1" ]; then
     # shellcheck disable=SC2086
-    "${YUM[@]}" "${OPTS[@]}" clean all
+    "${YUM[@]}" clean all
```

Comment:
This won't work this easily. `$OPTS` contains `--installroot` etc, without them it will clean the wrong cache.